odoo/tools/xml_utils.py

```diff
@@ -169,27 +168,19 @@ def load_xsd_files_from_url(env, url, file_name, force_reload=False,
     :param odoo.api.Environment env: environment of calling module
     :param str url: URL of XSD file/ZIP archive
     :param str file_name: the name given to the XSD attachment
-    :param bool force_reload: if True, reload the attachment from URL, even if it is already cached
+    :param bool force_reload: Deprecated.
     :param int request_max_timeout: maximum time (in seconds) before the request times out
     :param str xsd_name_prefix: if provided, will be added as a prefix to every XSD file name
     :param list | str xsd_names_filter: if provided, will only save the XSD files with these names
     :param func modify_xsd_content: function that takes the xsd content as argument and returns a modified version of it
     :rtype: odoo.api.ir.attachment | bool
-    :return: the main attachment or False if an error occurred (see warning logs)
+    :return: every XSD attachment created/fetched or False if an error occurred (see warning logs)
     """
     if not url.endswith(('.xsd', '.zip')):
         _logger.warning("The given URL (%s) needs to lead to an XSD file or a ZIP archive", url)
```

Comment:
@jco-odoo I have an issue there because in Gipuzkoa the link to the zip is
https://www.gipuzkoa.eus/documents/2456431/13761107/Esquemas+de+archivos+XSD+de+env%C3%ADo+y+anulaci%C3%B3n+de+factura_1_2.zip/2d116f8e-4d3a-bff0-7b03-df1cbb07ec52
And removing the hash at the end yields a 404 so we're not able to download the XSDs.
Just mentioning it in case we want to fix that as well :crossed_fingers: 
Nice PR anyway !